jekyll/_cci2/first-steps.adoc

```diff
@@ -10,62 +10,101 @@ contentTags:
 
-NOTE: The full set of documentation for integrating GitLab with CircleCI can be found on the link:/docs/gitlab-integration[GitLab integration page].
+[CAUTION] 
+====
+You must follow the link from an invite email in order to join an existing CircleCI organization. If you attempt to sign up directly from the link:https://circleci.com/signup/[Sign Up] page, you will be creating a new organization. 
 
-[#email-signup]
-== Sign up with email
+If you have not received an invite, please contact your org administrator.
+====
 
-. Click link:https://circleci.com/signup/[**Sign Up with Email**].
+. To create an account for free with CircleCI, follow the link in your invite email and enter your email address and password. 
 
-. Enter your email address, and then set a secure password for your CircleCI account. A verification email is sent to the email address provided.
+. On the next screen, respond to prompts that best describe your role and your engineering organization.
 
-. If you do not want to connect to your code and only wish to continue with the email signup, click **Cancel**. You will be taken to a screen where you can respond to prompts that best describe your role and your engineering organization.
+. Finally, you will be taken to your organization's pipelines dashboard in the CircleCI web app. From here, you can follow projects or trigger pipelines in projects to which you have access in your organization, or set up new projects.
 
-. Explore some example projects within the CircleCI app if you do not want to connect to your code at this time. You can take a look at a popular open source project building on CircleCI (link:https://app.circleci.com/pipelines/github/facebook/react[React by Facebook]), or one of our own sample projects: a link:https://app.circleci.com/pipelines/github/CircleCI-Public/sample-javascript-cfd/[sample JavaScript app], and a link:https://app.circleci.com/pipelines/github/CircleCI-Public/sample-python-cfd/[sample Python app].
+Note that if you are setting up a project for the first time, you may need to authenticate with your VCS provider. Once you have completed a one-time authentication, you will be able to set up subsequent projects in CircleCI more quickly. Refer to the xref:create-project#[Creating a Project] guide for more information. 
 
-You will be able to start exploring features such as link:/docs/pipelines[pipelines] and link:/docs/workflows[workflows]. The **Dashboard**, **Projects**, **Organization Settings**, and **Plan** pages are not available until you connect your code.
+The full set of documentation for integrating GitHub, Bitbucket, or GitLab projects are available as follows:
 
-When you are ready, you can connect to your GitHub, BitBucket, or GitLab accounts from the CircleCI web app.
+- xref:github-apps-integration#[GitHub App integration]
+- xref:bitbucket-integration#[Bitbucket integration]
+- xref:gitlab-integration#[GitLab integration]
 
 [#terms]
 == Terms
```

Comment:
I think this section needs checking for standalone projects maybe as the permissions are so different @sfranks01 ?